packages/keystatic/src/component-blocks/cloud-image.tsx

```diff
@@ -2,6 +2,8 @@ import { component } from '..';
 import { CloudImagePreview } from './cloud-image-preview';
 import { cloudImageSchema } from './cloud-image-schema';
 
+export type { CloudImageProps } from './cloud-image-preview';
```

Comment:
Needs to be re-exported from `packages/keystatic/src/component-blocks/index.tsx`